api/src/controllers/website.js

```diff
@@ -135,8 +135,27 @@ function deleteWebPage(req, res, next) {
     .catch((err) => next(fillErrorObject(500, 'Server error', [err])));
 }
 
+async function updatePublicationOptions(req, res, next){ // eslint-disable-line no-unused-vars
+  const updatedPubOptions = req.body; 
```

Comment:
The SA might pick this up when you merge in latest changes from `main`, but: https://eslint.org/docs/rules/prefer-destructuring